back/engines/commercial/multi_tenancy/app/services/multi_tenancy/templates/tenant_serializer.rb

```diff
@@ -41,7 +41,7 @@ def serialize_models
           AreasProject => serialize_records(AreasProject),
           AreasStaticPage => serialize_records(AreasStaticPage),
           Basket => serialize_records(Basket),
-          ContentBuilder::Layout => serialize_records(ContentBuilder::Layout),
+          ContentBuilder::Layout => serialize_records(ContentBuilder::Layout.where.not(content_buildable_type: 'HomePage')),
```

Comment:
Why not?

Reply:
This is because we include the homepage layout as nested attributes instead. The reason to do it that way is to avoid creating a default layout before creation of the homepage, which would result in the creation of two homepage layouts.